index.html

```diff
@@ -13,53 +13,70 @@
-          Glad to see you!</h2>
-          <p class="landing__text">
-            I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
-          </p>
-          <div><a href="#" class="landing__link">
-           let's connect
-          </a></div>
-          <div class="landing__social-media">
-            <ul>
-              <li><a href="#">
-              <object  class="landing__icon" data="/img/twitter.svg" height="20" width="20"></object>
-              </a></li>
-              <li><a href="#">
-                <object class="landing__icon" data="/img/Linkedin_icon.svg" height="20" width="20"></object>
-              </a></li>
-              <li><a href="#">
-                <object class="landing__icon" data="/img/medium.svg" height="20" width="20"></object>
-              </a></li>
-              <li><a href="#">
-                <object class="landing__icon" data="/img/github.svg" height="20" width="20"></object>
-              </a></li>
-              <li><a href="#">
-                <object class="landing__icon" data="/img/hands.svg" height="20" width="20"></object>
-              </a></li>
-            </ul>
-      </div>
+        <div class="landing__second-wrapper">
+          <h2 class="landing__header header">
+            <span>I'm Mariane</span>
+            <span>Glad to see you!</span>
+
+          </h2>
+            <p class="landing__text">
+              I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+            </p>
+            <div><a href="#" class="landing__link">
+             let's connect
+            </a></div>
+            <div class="landing__social-media">
+              <ul>
+                <li><a href="#">
+                <object  class="landing__icon" data="/img/twitter.svg" height="20" width="20"></object>
+                </a></li>
+                <li><a href="#">
+                  <object class="landing__icon" data="/img/Linkedin_icon.svg" height="20" width="20"></object>
+                </a></li>
+                <li><a href="#">
+                  <object class="landing__icon" data="/img/medium.svg" height="20" width="20"></object>
+                </a></li>
+                <li><a href="#">
+                  <object class="landing__icon" data="/img/github.svg" height="20" width="20"></object>
+                </a></li>
+                <li><a href="#">
+                  <object class="landing__icon" data="/img/hands.svg" height="20" width="20"></object>
+                </a></li>
+              </ul>
+            </div>
+        </div>
       </div>
     </section>
   </div>
 
   <!-- works -->
-  <div class="works">
+  <div class="works" id="portfolio-id">
```

Comment:
- [ ] [REQUIRED] It's better use `<section>` element instead of `<div>` element for semantic purposes ♻️.

Reply:
@MrRamoun  Thank you for your detailed review :slightly_smiling_face:  I found that a section must contain a `<header>` tag. Given this section displays different row elements It has no direct child header

Reply:
Hi @SirriCelles,

Kindly apply the changes as requested by the previous reviewer. Not only it is part of the project requirements but also it is more semantic to divide different sections of the page with a `section` tag. I'm not quite sure where you found that a section must have a `header`, but it is not quite correct. There must only be one header element on a page and it doesn't have to be in a `section` tag.

Reply:
Great job! 👍🏾 👏🏾 

Kindly request a re-review when you are ready, I've already submitted your work for required changes. 🙏🏾